### DISCUSSION TITLE
Could I pick only one editor when add a new page?

### DISCUSSION THREAD

**Discussion by NangeGao** [requarks/wiki#6430] [Help / Questions] [answered]
![image](https://github.com/requarks/wiki/assets/6187897/5c2ebc05-fcfc-40ba-b6d5-f008d8ec7fab)

Now, you see, there are 5 choices for users. I want all users write new articles by Markdown format. What should I do, pls?

**Comment by NGPixel** [comment, marked as answer]
This is coming in v3.

**Reply by NangeGao** [reply]
Thank you. May I know when is v3 coming?
